src/Symfony/Component/Cache/composer.json

```diff
@@ -31,11 +31,14 @@
         "doctrine/cache": "~1.6",
         "doctrine/dbal": "~2.5",
         "predis/predis": "~1.0",
-        "symfony/var-dumper": "^4.1.1"
+        "symfony/var-dumper": "^4.1.1",
+        "symfony/dependency-injection": "~4.2",
+        "symfony/config": "~4.2"
```

Comment:
let's insert in alpha order as much as possible (same below)
is "config" really required?

Reply:
This was the issue I got with `CachePoolPrunerPassTest`.
If I don't have `Config` component, `ContainerBuilder` won't have the `$trackResources` variable as `true` and the `CachePoolPrunerPassTest::testCompilerPassThrowsOnInvalidDefinitionClass` test won't pass at all.